halconfig/settings.js

```diff
@@ -105,7 +105,7 @@ window.spinnakerSettings = {
   authEnabled: authEnabled,
   authTtl: 600000,
   gitSources: ['stash', 'github', 'bitbucket'],
-  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins'],
+  triggerTypes: ['git', 'pipeline', 'docker', 'cron', 'jenkins', 'travis'],
```

Comment:
@danielpeach This line does not exist in that file; should I add it?
https://github.com/spinnaker/spinnaker/blob/master/config/settings.js#L122